### DISCUSSION TITLE
Is it possible to hide the port for fronted?

### DISCUSSION THREAD

**Discussion by resuvelt** [walkor/Workerman#860] [Q&A]
Hello, @walkor!

Is there a possibility of port routing, for example, instead of ws://localhost:number_post on ws://localhost/random_text if so, how do I do it?

**Comment by walkor** [comment]
Use nginx as proxy.
```
location /ws
  {
    proxy_pass http://127.0.0.1:8282;
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "Upgrade";
    proxy_set_header X-Real-IP $remote_addr;
  }
```
Then connect `ws://localhost/ws` instead of `ws://localhost:port`
